protocol/linkpreview_unfurler_image.go

```diff
@@ -105,6 +106,7 @@ func (u *ImageUnfurler) Unfurl() (*common.LinkPreview, error) {
 		return preview, fmt.Errorf("could not build data URI url='%s': %w", u.url.String(), err)
 	}
 
+	preview.Title = path.Base(u.url.Path)
```

Comment:
@igor-sirotin, just a question, from the original issue, it says the file name should be used as the title, but here we're using the URL path. It's not the same, but I get that for many URLs the best we can do is get the path.

But taking a look a this example:

https://images.all-free-download.com/images/graphicwebp/gmorning_cat_1_554356.webp

The correct title would be `gmorning_cat_1_554356.webp`, not `images/graphicwebp/gmorning_cat_1_554356.webp` (I'm ignoring the text truncation part for this example.)

Reply:
Oh I totally missed the `Base` call 🦀 Thanks